src/tools/rbd/action/Trash.cc

```diff
@@ -36,8 +36,8 @@ void get_move_arguments(po::options_description *positional,
                         po::options_description *options) {
   at::add_image_spec_options(positional, options, at::ARGUMENT_MODIFIER_NONE);
   options->add_options()
-    (at::DELAY.c_str(), po::value<uint64_t>(),
-     "time delay in seconds until effectively remove the image");
+    (at::EXPIRES_IN.c_str(), po::value<std::string>()->default_value("1 week"),
```

Comment:
Nit: why not default to "now" since "1 week" otherwise seems arbitrary?

Reply:
It is still in discussion, we wanted some safe date in case someone mistakenly trashes an image and a trash purge cron job runs after a second and deletes it (and then the image can't be restored). It can be changed to a "day", or "now" if it makes more sense.